### ISSUE TITLE
Focus on Invalid field on Submit

### ISSUE THREAD

**Issue by ShahanaFarooqui** [ngx-formly/ngx-formly#1413]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[X] support request
```
**Current behavior**
I want to reset the focus on invalid field if the validation failed of that field. I am validating all the fields on form submit event. Currently, i can only set focus with expressionProperties once but unable to reset it after validation.

**Expected behavior**
Invalid field should get the focus after validation failed.

**Minimal reproduction of the problem with instructions**
I tried to toggle focus on single field in below example. I tried to set the focus from one field to another but that is also not working.
https://stackblitz.com/edit/angular-tpbjwz

**Please tell us about your environment:**
* **Angular version:**
  Angular Version - 6.1.X
  ngx-formly/core - 4.7.2
* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

In all Browsers irrespective of the version.
* **Language:** [all | TypeScript X.X | ES6/7 | ES5] - ES7
* **Node (for AoT issues):** `node --version` =10

**Comment by aitboudad** [comment]
fixed in `5.0.0-rc.1`

**Comment by aitboudad** [comment]
you should try with `5.0.0-rc.3` ( `rc.1` is broken)
